namespaces/docker.yml

```diff
@@ -9,3 +9,103 @@ namespace:
     type: string
     description: >
       Docker container id
+  - name: event
+    type: group
+    description: >
+      Event emitted by docker daemon that describes some activity that occurred in the past
+    fields:
+    - name: action
+      type: string
+      example: create
+      description: >
+        What activity occurred
+    - name: status
+      type: string
+      example: create
```

Comment:
Will the docker daemon emit `create` for the status at the same time emitting `create` for the action above?  Perhaps having a few example events in the group description would be useful.